### ISSUE TITLE
SMTP Pro and Office 365

### ISSUE THREAD

**Issue by ZenAdm** [aschroder/Magento-SMTP-Pro-Email-Extension#210]
Hi Ashley, 

Magento 1.9.3.3
SMTP Pro 2.0.7

We use every day Microsoft Office 365 (mail server in Microsoft's cloud).

**SMTP Pro configuration** 
Connexion Mail : custom smtp
Authentification : Plain (try in connexion too)
Username : courrier@welfarm.fr
Password : my_password
Host : smtp.office365.com
Port : 587 (try in 25 too)	
Sécurité : SSL TLS

**Error message** 
```
Résultats de SMTP Pro Auto-test
Sending test email to your contact form address: courrier@welfarm.fr from: courrier@welfarm.fr. Impossible d'envoyer l'email de test.
Message d'exception était: 5.7.4 Unrecognized authentication type [DB6PR07CA0075.eurprd07.prod.outlook.com] 
S'il vous plaît vérifier le mode d'emploi pour les messages d'erreur fréquents et leurs solutions.
Les modèles par défaut existent.
Les communications par courriel sont activés.
Tables de base de données requises existent.
Testing failed, please review the reported problems and if you need further help visit the support page or contact me via support@aschroder.com for support.
```

Have you an idea ? 
TY
ZenAdm

**Comment by ZenAdm** [comment]
https://ecommerce.aheadworks.com/kbase/authentication-with-microsoft-smtp-server.html
-> It's an authentification problem, office365 want a NTLM auth ...

**Comment by ZenAdm** [comment]
UP

**Comment by ZenAdm** [comment]
UP

**Comment by xqiu** [comment]
smtp works for me for office 365,   
The only different setting I have is: Authentification : Login
Note, if it still doesn't work, try to set the "Store Email Addresses"'s sender email the same as the email you setup here.

**Comment by ajaygohel** [comment]
I'm also using Custom SMTP with office 365 host.
As suggested by 'xqiu' i tried with same "Store Email Addresses" as a sender with SMTP configuration setting. Hopefully it's working, But when i use different "Store Email Addresses" after then mail is not going to send. Is there any way that i can use different sender email address as well?

**Comment by vernard** [comment]
Yes, it is indeed possible to use a different sender email address. However, it just says "on behalf of..." instead of the actual different sender email address. See the attached image below:

![image](https://user-images.githubusercontent.com/6733673/33305529-7ccc38a4-d449-11e7-8e25-531ba7531578.png)

**Comment by ZenAdm** [comment]
Impossible to install SMTP Pro from https://marketplace.magento.com/ (magento connect), so impossible to try xqiu idea.

**Comment by xqiu** [comment]
ZenAdm, installing open source extensions by downloading this source, and copy app folder over...

**Comment by Autofaszination** [comment]
the test mail works fine. but if I order something I dont get the mails. order and the bill doesnt work

**Comment by Anantkprajapati** [comment]
Hello Guys

Is there any solution for this. I am also facing same issue

Thank you

**Comment by kkrieger85** [comment]
@Anantkprajapati Please be sure that `smtppro/general/smtp_username` and `trans_email/ident_general/email` (Store Emails) are the same.

**Comment by Anantkprajapati** [comment]
Hello @kkrieger85 - it's fixed now. May be it cause of the office365 issue.
